spec.bs

```diff
@@ -292,9 +294,10 @@ observable.subscribe({}, {signal: outerController.signal});
 
     1. Set |subscriber|'s [=Subscriber/active=] boolean to false.
 
-    1. [=AbortSignal/Signal abort=] |subscriber|'s [=Subscriber/subscription controller=].
+    1. [=AbortSignal/Signal abort=] |subscriber|'s [=Subscriber/subscription controller=]
+       with |reason|, if it is given.
 
-       Issue: Abort with an appropriate abort reason.
+       Issue: Abort with an appropriate abort reason if none given.
```

Comment:
I don't think this is needed at all now. We don't need to supply a "fallback" default reason if none was provided.